terms/experimentalData/assayTarget.json

```diff
@@ -394,10 +404,15 @@
             "source": ""
         },
         {
-            "const": "SOX8",
+            "const": "SMC3",
```

Comment:
It looks like "SOX8" was replaced with "SMC3". Did this need to be in addition, not in replacement?